### PR TITLE
feat: filter recent contributions array by last updated time

### LINKED ISSUE #682: Feature: Sort PRs based on updated_at
### Type of feature

🍕 Feature

### Current behavior

The current implementation returns PRs based on when they opened, but should instead order based on the last commit. 

This is a screenshot of @SunGoldTech's PR Deploy preview. The PR should be on the top and should say `1d:` or similar to show the last commit was within the last day. 

Screen Shot 2022-12-19 at 9 58 14 AM

### Suggested solution

Note that the 12d prepended to the PR title is the same 12s as in the column. These dates should be different. The column is the date the PR was created.

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by SunGoldTech**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
This PR:

- Sorts the `contributorPRs` array data by the most recent `updated_at` time value
- Updates the last updated time value

## Related Tickets & Documents
Fixes #682 

## Mobile & Desktop Screenshots/Recordings
image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Review by 0-vortex — APPROVED** [review]
LGTM! 👍

**Comment by brandonroberts** [comment]
@SunGoldTech the data should already be sorted from the API. We need to update this line https://github.com/open-sauced/insights/blob/beta/components/molecules/LatestPrTableRow/latest-pr-table-row.tsx#L59 to use the `updated_at` in relative days

**Comment by SunGoldTech** [comment]
Ok

**Comment by bdougie** [comment]
These dates should be different. Left is last commit date. Right is opened PR
208699895-a55566e7-fab5-4d82-983c-2fe2d1e20a65

**Comment by SunGoldTech** [comment]
> These dates should be different. Left is last commit date. Right is opened PR 

This has been fixed in my last commit

### Currently
image

File: components/molecules/LatestPrTableRow/latest-pr-table-row.tsx [hunk 1]
```diff
@@ -56,7 +58,7 @@ const LatestPrTableRow = ({
             <VscGitPullRequestDraft title="Draft Pull Request" />
           </IconContext.Provider>
         )}
-        <Text>{calcDistanceFromToday(new Date(parseInt(prIssuedTime, 10)))}</Text>
+        <Text title="last commit date">{calcDistanceFromToday(new Date(parseInt(prUpdatedTime, 10)))}</Text>
```
Comment (brandonroberts) [inline components/molecules/LatestPrTableRow/latest-pr-table-row.tsx:61]:
```suggestion
        <Text title="updated date">{calcDistanceFromToday(new Date(parseInt(prUpdatedTime, 10)))}</Text>
```

File: components/molecules/ContributorTable/contributor-table.tsx [hunk 2]
```diff
@@ -24,12 +24,14 @@ interface CardTableProps {
...
-  return data.length > 0 ? (
+  const sortedPr = data?.sort((a: any, b: any) => parseInt(b.updated_at) - parseInt(a.updated_at));
+
+  return sortedPr.length > 0 ? (
     <>
       <div className="flex flex-col">
         <LatestPrTableHeader isHoverCard={isHoverCard} />
         <div className="flex flex-col gap-0.5">
-          {data.map(
+          {sortedPr.map(
```
Comment (brandonroberts) [inline components/molecules/ContributorTable/contributor-table.tsx:34]: As a note, `sort()` mutates the original array, so a separate `sortedPr` variable isn't needed

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: components/molecules/ContributorTable/contributor-table.tsx (hunk 2 above)
Comment (brandonroberts) [reply components/molecules/ContributorTable/contributor-table.tsx:34]: If you want to use a new variable, you can create a copy of the original array using spread syntax or the `slice()` method on the  array

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**